src/main/java/focusedCrawler/link/classifier/LNClassifier.java

```diff
@@ -55,37 +66,25 @@ public static LNClassifier create(String featureFilePath,
 	
 	public static LNClassifier create(String[] attributes, String[] classValues,
 	                                  String modelFilePath, StopList stoplist) {
-	    weka.core.FastVector vectorAtt = new weka.core.FastVector();
-	    for (int i = 0; i < attributes.length; i++) {
-	        vectorAtt.addElement(new weka.core.Attribute(attributes[i]));
-	    }
-	    weka.core.FastVector classAtt = new weka.core.FastVector();
-	    for (int i = 0; i < classValues.length; i++) {
-	        classAtt.addElement(classValues[i]);
-	    }
-	    vectorAtt.addElement(new weka.core.Attribute("class", classAtt));
-	    Instances insts = new Instances("link_classification", vectorAtt, 1);
-	    insts.setClassIndex(attributes.length);
+//	    weka.core.FastVector vectorAtt = new weka.core.FastVector();

```

Comment:
Please remove all old, comment code when the PR is done.

Reply:
Sure.. Will do.